### PR TITLE
Fix cumulus companion CI job

### PR DESCRIPTION
**Pull request by bkchr**
The fix is here https://github.com/paritytech/pipeline-scripts/pull/12

### CODE REVIEW

**Review by thiolliere — APPROVED** [review]
seems good to me

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -292,19 +292,21 @@ build-malus:
...
-        "--branch=$PIPELINE_SCRIPTS_TAG"
+        "--branch=bkchr-update-cargo"
         https://github.com/paritytech/pipeline-scripts
     - ./pipeline-scripts/check_dependent_project.sh
         paritytech
         polkadot
         --polkadot
         "$DEPENDENT_REPO"
         "$GITHUB_PR_TOKEN"
+        $CARGO_UPDATE_CRATES
```
Comment (TriplEight) [inline .gitlab-ci.yml:303]:
```suggestion
        "$CARGO_UPDATE_CRATES"
```
let's be consistent

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -292,19 +292,21 @@ build-malus:
   script:
     - git clone
         --depth=1
-        "--branch=$PIPELINE_SCRIPTS_TAG"
+        "--branch=bkchr-update-cargo"
```
Comment (TriplEight) [inline .gitlab-ci.yml:295]:
```suggestion
        "--branch=$PIPELINE_SCRIPTS_TAG"
```

**Review by TriplEight — CHANGES_REQUESTED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (bkchr) [reply .gitlab-ci.yml:303]: This breaks the script.

Comment (TriplEight) [reply .gitlab-ci.yml:303]: ah that's a space-separated list, better `{}` then

**Review by TriplEight — APPROVED** [review]

**Pull request merged**